pkg/hooks/hooks_test.go

```diff
@@ -77,3 +79,44 @@ func testManifest() pods.Manifest {
 	builder.SetID(podId)
 	return builder.GetManifest()
 }
+
+func TestHookRunWithTimeout(t *testing.T) {
```

Comment:
I remember the initial version had a run that would succeed (2 second timeout, 1 second sleep) - did we consider this not worth keeping? I might have liked it! Perhaps just a simple run of `true` with some nonzero timeout which should end quickly (I believe that was suggested before)